src/include/daos_api.h

```diff
@@ -220,11 +220,149 @@ daos_prop_alloc(uint32_t entries_nr);
 /**
  * Free the DAOS properties.
  *
- * \param[int]	prop	properties to be freed.
+ * \param[in]	prop	properties to be freed.
  */
 void
 daos_prop_free(daos_prop_t *prop);
 
+/**
+ * Allocate an DAOS Access Control List.
+ *
+ * \param[in]	aces		Array of pointers to ACEs to be put in the ACL.
+ * \param[in]	num_aces	Number of ACEs in array
+ *
+ * \return	allocated daos_acl pointer, NULL if failed
+ */
+struct daos_acl *
+daos_acl_alloc(struct daos_ace *aces[], uint16_t num_aces);
+
+/**
+ * Free a DAOS Access Control List.
+ *
+ * \param[in]	acl	ACL pointer to be freed
+ */
+void
+daos_acl_free(struct daos_acl *acl);
+
+/**
+ * Get the first Access Control Entry in the Access Control List, for iterating
+ * over the list.
+ *
+ * \param[in]	acl	ACL to traverse
+ *
+ * \return	Pointer to the first ACE in the ACL, or NULL if none exists
+ */
+struct daos_ace *
+daos_acl_get_first_ace(struct daos_acl *acl);
```

Comment:
Could this method be == daos_acl_get_next_ace(..., NULL)?

Reply:
agreed.

Reply:
Will remove